tests/sentry/issues/test_run.py

```diff
@@ -20,13 +22,15 @@
 
 
 # need to shut down the connections in the thread for tests to pass
-def process_occurrence_group_with_shutdown(*args, **kwargs):
-    process_occurrence_group(*args, **kwargs)
+def process_occurrence_group_with_shutdown(items: list[Mapping[str, Any]]) -> None:
+    process_occurrence_group(items)
```

Comment:
Feels like only `**kwargs` can be replaced by the `list[Mapping[str, Any]]` type and `*args?` cannot since there's  no keywords? Or is there some Python/mypy magic happening behind the scenes that makes this okay?

Reply:
`**kwargs` could have only been passed as `items=...` before as the underlying signature of `process_occurrence_group` is `(list[Mapping[str, Any]]) -> None` -- so this just makes the signature match explicitly

with `*args, **kwargs` mypy infers that as "this is a callable that could take any number of positional or named arguments" -- now it's explicitly "one possibly-named argument possibly positional argument named `items` of type ..."